genesyscloud/tfexporter/resource_genesyscloud_tf_export_test.go

```diff
@@ -680,10 +680,10 @@ func TestAccResourceTfExportUserPromptExportAudioFile(t *testing.T) {
 					falseValue,
 					falseValue,
 				) + gcloud.GenerateUserPromptResource(&gcloud.UserPromptStruct{
-					userPromptResourceId,
-					userPromptName,
-					strconv.Quote(userPromptDescription),
-					userPromptResources,
+					ResourceID:  userPromptResourceId,
```

Comment:
Refactored this because Go best practice is to explicitly have key names in your struct definition.  Caught this via go vet.